src/components/common/BarChartSkeleton/BarChartSkeleton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid, Skeleton } from '@mui/material';
+
+interface BarChartSkeletonProps {
+  bars: number;
+}
+
+export const BarChartSkeleton: React.FC<BarChartSkeletonProps> = ({ bars }) => (
+  <Grid
+    container
+    justifyContent="space-between"
+    alignItems="flex-end"
+    height="100%"
+    data-testid="BarChartSkeleton"
+  >
+    {new Array(bars).fill(undefined).map((_, index) => (
+      <Skeleton
+        key={index}
```

Comment:
Keys don't need to be unique across the page. They just need to be unique inside a single `.map()` call.

https://react.dev/learn/rendering-lists#rules-of-keys

Reply:
You are correct. I was getting mixed up with not using the index as the key, but in this case, this is fine, as even if the skeletons change orders, it will still work, and it won't matter.